### PR TITLE
doc: Add guide for MikroTik Routerboard

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

References #1140 

#### Changes

- Add connection guide

#### Notes for Reviewers

There are also Windows/macOS GUIs but the smarpthone apps are the easiest way to configure this gateway.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [ ] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title: "MikroTik Routerboard wAP LoRa8 kit"
+description: ""
+weight: 1
+---
+
+MikroTik Routerboard wAP LoRa8 kit is a LoRaWAN gateway that contains a pre-installed UDP packet forwarder and an outdoor weatherproof wireless access point with 2.4 GHz WLAN interface and Ethernet port that could be used as a backend. It's technical specifications can be found in [the official documentation](https://mikrotik.com/product/wap_lora8_kit). 
+
+There are multiple interfaces to configure the gateway parameters. This page guides you to connect it to {{% tts %}} using the [MikroTik Mobile App](https://mikrotik.com/mobile_app) for Android/iOS.
+
+## Physical connections
```
Comment (johanstokking) [inline doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md]: Title case for titles

File: doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{{< figure src="mikrotik-routerboard-lora-disable.jpeg" alt="MikroTik Routerboard wAP LoRa8 kit LoRa disable" >}}
+
+Back in the **LoRa** section, select the **Servers** section. Select the **+** button to add a new server. 
+
+Edit the server parameters.
+
+1. **Name**: A distinct name 
+2. **Address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
+3. **Up port**: UDP upstream port of the Gateway Server, typically 1700.
+4. **Down port**: UDP downstream port of the Gateway Server, typically 1700.
```
Comment (johanstokking) [inline doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md]: How about the frequency plan? Can we fetch this from GCS?

**Comment by KrishnaIyer** [comment]
Blocked on firmware update to support changing frequency plans by MikroTik.

**Comment by johanstokking** [comment]
Do we have an update from MikroTik?

**Comment by KrishnaIyer** [comment]
None unfortunately.

**Comment by KrishnaIyer** [comment]
@benolayinka: Please rebase on `master` and force-push seems like there's a mistake in the way your updated this branch.

File: doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
```
Comment (rvolosatovs) [inline doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md:105]: In my case I also had to change the server type to "private" to make it work- it's "public" by default

Comment (johanstokking) [reply doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md:105]: Hmm, can you guys check what that means?

On MultiTech gateways, it has to be the other way around. Private has a different sync word than public, and the LoRaWAN ecosystem uses the public one. Again, in MultiTech terms. So. I'd be curious to what it means on MikroTik and if it's opposite.

Comment (benolayinka) [reply doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md:105]: I didn't have to change it - left public and connected to the enterprise stack. I never routed any packets through it though.

I can't find any documentation from Mikrotik about what it means. I'll send an email.

Comment (benolayinka) [reply doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md:105]: Edit: nevermind, mine was set to private by default.

Comment (benolayinka) [reply doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md:105]: @johanstokking from the [Mikrotik docs](https://help.mikrotik.com/docs/display/ROS/General+Properties)

network (private \| public; Default: public) | Whether sync word should (network=private) or should not (network=public) be used.
-- | --

Comment (laurensslats) [reply doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md:104]: I tried both public and private, both worked

Comment (benolayinka) [reply doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md]: @laurensslats did you route packets?

**Pull request merged**